samples/debug/memfault/README.rst

```diff
@@ -158,7 +155,9 @@ Testing
 Before testing, ensure that your device is configured with the project key of your Memfault project.
 |test_sample|
 
-1. |connect_terminal_ANSI|
+
+1.  |connect_terminal_ANSI|
+There is also possible to connect with the kit using the nrf Connect for VS Code extension and `connect to a serial nrf Serial Terminal. <How to connect to the terminal>`
```

Comment:
We have NCSDK-20386 Jira open to change the recommended serial terminal to VSC terminal in all samples and applications. May be remove this for now as this be consider in this Jira.